Modules/hpe3par_host.py

```diff
@@ -538,6 +539,20 @@ def add_fc_path_to_host(
             {})
     try:
         client_obj.login(storage_system_username, storage_system_password)
+
+        # check if wwn is already assigned
+        client_obj.setSSHOptions(storage_system_ip, storage_system_username, storage_system_password)
+
+        for fc_wwn in host_fc_wwns:
+            host_name_3par = client_obj.findHost(wwn=fc_wwn)
+
+            if host_name_3par:
+                if host_name == host_name_3par:
+                    return (True, False, "WWN %s is already assigned to host" % fc_wwn, {})
+                else:
```

Comment:
Do we really need the else part? This error msg will be propogated from the array.

Reply:
there is difference.
If host_name is same then it means customer is performing operation on same host again.
So True is returned, so that remaining operations can continue.
If host_name is different then it means the wwn is used by another host & error should be reported.
So False is returned.

Reply:
But if we don't have the else part, wont the playbook give False and propogate same msg?